torch/nn/quantizable/modules/activation.py

```diff
@@ -0,0 +1,439 @@
+import torch
```

Comment:
nit on the filename, is `activation.py` the best filename for it?  I would have expected something like `multiheadattention.py`, etc

Reply:
MHA is a type of activation by convention -- it's more of a head (imho), but pytorch does not have a "head" subgroup for the layers, so it is put in the activation group